Program_1.0.py

```diff
@@ -40,37 +87,33 @@ def comparing_flags(input_flag):
 
 
 def phrase_with_tag(country, tagged_word):
+    if db.countries.find({'name': country}).count() == 0:
+        saving_country_to_db(country)
     whole_country_summary = TextBlob(db.countries.find_one({'name': country})['Summary']).sentences
     tagged_phrases = []
     for sentence in whole_country_summary:
         if sentence.find(tagged_word) != -1:
             sentence = str(sentence)
             tagged_phrases.append(sentence)
-    return tagged_phrases
+    for phrase in tagged_phrases:
+        return phrase
 
 
-##################################################################
+def downloading_content(chosen_data):
+    country_name = key_words_from_content(chosen_data)
+    if chosen_data.find('tag') != -1:
+        tag = re.search('tag\((.*)\)', chosen_data)
+        tag = tag.group(1)
+        return phrase_with_tag(country_name, tag)
+    elif chosen_data.find('getflag') != -1:
+        return 'https://en.wikipedia.org/wiki/File:Flag_of_' + country_name + '.svg'
+    elif chosen_data.find('country') != -1:
+        return pprint.pprint(whole_country_summary(country_name))
+    elif chosen_data.find('checkflag') != -1:
+        urllib.request.urlretrieve(country_name, 'flag.png')
+        flag = Image.open('flag.png').convert('RGB')
+        return comparing_flags(flag)
```

Comment:
do całej funkcji. to jest takie very nice to have tj. żeby wszystko działało na zasadzie if: return, elsif:return etc. i cała logika działa się w wewnątrz

Reply:
i minor ale jestem przeciwnikiem BRAKU elsa

Reply:
Nawet jeśli miałoby być else: pass?

Reply:
dałbym else: print("Does not recognize command") albo coś bo potem łatwo się takie coś debuguje